roles/kafka_broker/defaults/main.yml

```diff
@@ -13,6 +13,25 @@ kafka_broker_max_log_files: 10
 ### Max size of a log file generated by Kafka Broker. Only honored if kafka_broker_custom_log4j: true
 kafka_broker_log_file_size: 100MB
 
+### Boolean to enable logredactor for kafka_broker. Defaults to logredactor_enabled. Works only when kafka_broker_custom_log4j is set to true
+kafka_broker_logredactor_enabled: "{{ logredactor_enabled }}"
+
+### Location of the kafka_broker logredactor rules file on the control node. This'll be copied to kafka_broker_logredactor_rule_path on the kafka_broker node. 
+kafka_broker_logredactor_rule_path_local: "{{ logredactor_rule_path_local }}"
+
+### Full path and name of the rules file on kafka_broker node. This is the location which will be referenced in the log4j property file when defining the redactor appender.
+kafka_broker_logredactor_rule_path: "{{ logredactor_rule_path }}"
+
+### URL address that contains or redirects to the redaction rules. If not "", this will be used to fetch rules. kafka_broker_logredactor_rule_path will be ignored.
+kafka_broker_logredactor_rule_url: "{{ logredactor_rule_url }}"
+
+### If present, it's used to specify a time in ms for how often the file system or URL of the policy rules will be checked for changes. If set to 0 (default), the policy rules will only be read once at startup.
+kafka_broker_logredactor_policy_refresh_interval: "{{ logredactor_policy_refresh_interval }}"
+
+### List of loggers to redact. This is specified alongside the user defined redactor name and appenderRefs to be used in redactor definition.  
+kafka_broker_logredactor_logger_specs_list:
+  - { logger_name: "log4j.rootLogger", appenderRefs: "kafkaAppender", redactor_name: "redactor1" }
```

Comment:
Do we need to get the redactor names form user?

Reply:
Does the name have to be like `redactor[1..n]` . Can it not be like `Redactor_{annderRefs}`.

Reply:
If we have to build out the redactor name in the code, I have used this `redactor_{{ item.logger_name | replace ('.', '_') }}`

Reply:
Thanks!